src/containers/MyNdla/Arena/components/PostCard.tsx

```diff
@@ -71,11 +71,18 @@ const StyledTopContainer = styled.div`
 
 const StyledSwitch = styled(Switch)`
   align-self: flex-start;
+  border: 2px solid transparent;
+  border-radius: ${misc.borderRadius};
   padding: ${spacing.xsmall};
   ${mq.range({ until: breakpoints.desktop })} {
     align-self: flex-end;
     margin-bottom: ${spacing.small};
   }
+  &:focus,
+  &:focus-visible,
+  &:focus-within {
+    border-color: black;
```

Comment:
colors.brand.dark ? Blir ikke det fargen vi har brukt for det meste på fokus?

Reply:
Jo. Berre eg som blingsa litt.